.github/workflows/coverity.yml

```diff
@@ -0,0 +1,21 @@
+name: Coverity
+
+on:
+  pull_request:
+    branches:
+      - "*"
+  push:
+    branches:
+      - main
+
+jobs:
+  lint:
+    name: "Coverity scan"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Run coverify scan script
+        run: ./tools/coverity-scan.sh
```

Comment:
you can map secrets to the environment here using 
```
env: 
  SECRET_NAME: ${{ secret.SECRET_NAME }}
```

Reply:
I put this in workflow file, but got error "unrecognized named-value: 'secret'". I will check how to use the secret.
    env: 
      COVERITY_SCAN_PROJECT_NAME: ${{ secret.COVERITY_SCAN_PROJECT_NAME }}

Reply:
oh, maybe it should be secrets, but not secret. I will try.